site/app/controllers/forum/ForumController.php

```diff
@@ -388,14 +388,16 @@ public function publishThread(){
                 $metadata = json_encode(array(array('component' => 'forum', 'page' => 'view_thread', 'thread_id' => $thread_id)));
                 // notify on a new announcement
                 if ($announcement) {
-                    $content = "New Announcement: ".$thread_title;
-                    $event = ['component' => 'forum', 'metadata' => $metadata, 'content' => $content, 'subject' => 'New Announcement'];
+                    $subject = "New Announcement: ".Notification::textShortner($thread_title);
+                    $content = "An Instructor/TA made an announcement in the Submitty discussion forum:\n\n".$thread_post_content;
```

Comment:
Per Barb's suggestion here and in the else I think it would be a good idea to include the unshortened title at the beginning of the content in case vital information or the only information is in the title

I think in every other place the title is included in the content somehow but feel free to correct me